tests/php/tests/includes/rest-api/test_class.wp-job-manager-job-listings.php

```diff
@@ -156,6 +162,268 @@ public function test_get_job_listings_success_guest() {
 		$this->assertResponseStatus( $response, 200 );
 	}
 
+	/**
+	 * Tests to make sure public meta fields are exposed to guest users and private meta fields are hidden.
+	 */
+	public function test_guest_read_access_to_private_meta_fields() {
```

Comment:
Yay tests!! Another method name could be `test_guest_can_read_only_public_fields`

Reply:
Sounds good! 0a6a578df5a7772c80221c3268d173f9148897fc